.github/workflows/check-csv-ordering.yaml

```diff
@@ -0,0 +1,28 @@
+name: Check CSV order
+
+on:
+
+  push:
+    branches:
+      - ci-check-csv-ordering
+
+  pull_request:
+
+
+jobs:
+  check-order:
+    runs-on: [ubuntu-latest]
+
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Generate header-free version
+        run: tail -n +2 fate-product-list.csv > headerless.csv
+
+      - name: Generate sorted version
+        run: cat headerless.csv | LC_ALL=en_US.UTF-8 sort | head -c -1 > sorted.csv
```

Comment:
`sort` accepts a file to sort on the command line, so this should work:

```bash
LC_ALL=en_US.UTF-8 sort headerless.csv | head -c -1 > sorted.csv
```

Reply:
Applied.